llvm/include/llvm/CodeGen/FunctionLoweringInfo.h

```diff
@@ -212,6 +212,11 @@ class FunctionLoweringInfo {
     return ValueMap.count(V);
   }
 
+  MachineBasicBlock *getMBB(const BasicBlock *BB) const {
+    assert(BB->getNumber() < MBBMap.size() && "uninitialized MBBMap?");
+    return MBBMap[BB->getNumber()];
+  }
```

Comment:
Can you please pre-commit the addition and use of this method as a separate NFC change?

Reply:
#101893